bliss/encoder.py

```diff
@@ -137,6 +151,16 @@ def get_input_tensor(self, batch):
             ), "concat_psf_params specified but psf params not present in data"
             n, _, h, w = batch["images"].size()
             inputs.append(batch["psf_params"].view(n, 6, 1, 1).expand(n, 6, h, w))
+        if self.input_transform_params.get("z_score"):
+            assert (
+                batch["background"][0, 0].std() > 0
+            ), "Constant backgrounds not supported for multi-band encoding"
+            inputs[0] = (
+                batch["images"] - torch.mean(batch["images"], dim=(2, 3), keepdim=True)
+            ) / torch.std(batch["images"], dim=(2, 3), keepdim=True)
+            inputs[1] = (
+                batch["background"] - torch.mean(batch["background"], dim=(2, 3), keepdim=True)
+            ) / torch.std(batch["background"], dim=(2, 3), keepdim=True)
```

Comment:
The black autoformater strikes again! Will you try writing this with intermediate variables so it's a bit easier to read?

Reply:
Sounds good!